src/Core/src/Handlers/Button/ButtonHandler.Android.cs

```diff
@@ -123,7 +123,9 @@ public static Task MapImageSourceAsync(IButtonHandler handler, IImage image)
 
 		public override void PlatformArrange(Rect frame)
 		{
+			// The TextView might need an additional measurement pass at the final size
 			this.PrepareForTextViewArrange(frame);
```

Comment:
This is needed because TextView controls may be measured with an "AtMost" which means the measure is more "give me the smallest size". Because the text in a TextView is positioned using an absolute coordinate, it needs to be re-measured with the final sizes to get the new coordinates.